posts/2021-08-30-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: "Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mezarin
+author_github: https://github.com/mezarin
+seo-title: Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster - OpenLiberty.io
+seo-description: Speed up development by making it easier for application developers to bind operator managed services to applications.
+blog_description: "Speed up development by making it easier for application developers to bind operator managed services to applications"
+open-graph-image: https://openliberty.io/img/blog/ol-stack-jpa-app-db-bind-browser-data-entry.png
+---
+= Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster
+Edward Mezarina <https://github.com/mezarin>
+
+Binding your Microservice application to operator based services (e.g. databases, etc.) has now been made easier through automatic collection of service data, deployment of needed resources, and data sharing with the application.
+
+== Open Liberty devfile stack
```

Comment:
since this section and the next (Openshift Do) are the same as what you had for the Gradle devfile post, reccommend copying the text we edited there into this post